client/src/main/java/org/apache/rocketmq/client/ClientConfig.java

```diff
@@ -111,6 +111,8 @@ public class ClientConfig {
      */
     protected String traceTopic;
 
+    protected int maxPageSizeInGetMetadata = 2000;
```

Comment:
It may not be necessary to add this parameter to clientConfig; perhaps it would be better to allow the caller to set it based on their specific needs.